influxql/ast.go

```diff
@@ -683,6 +758,11 @@ func (s *DeleteStatement) String() string {
 	return s.String()
 }
 
+// RequiredPrivilege returns the privilege required to execute a DeleteStatement.
+func (s *DeleteStatement) RequiredPrivileges() ExecutionPrivileges {
+	return ExecutionPrivileges{{Name: "", Privilege: WritePrivilege,},}
```

Comment:
That's what this means.  Some statements (like this one) don't know what database they're going to be executed on so it just leaves the name blank.  The caller knows and will fill it in.  The reason to have the `Name` field is that some statements do specify the database and some statements require different privileges on different databases.  For example, calling `RequiredPrivileges` on a `SELECT ... INTO` may return multiple required privileges.  It might require read from `fooDB` and write to `barDB`.